scalastyle-config.xml

```diff
@@ -206,7 +206,7 @@ This file is divided into 3 sections:
       <parameter name="groups">java,scala,3rdParty,kyuubi</parameter>
       <parameter name="group.java">javax?\..*</parameter>
       <parameter name="group.scala">scala\..*</parameter>
-      <parameter name="group.3rdParty">(?!org\.apache\.kyuubi\.).*</parameter>
+<!--      <parameter name="group.3rdParty">(?!org\.apache\.kyuubi\.).*</parameter>-->
```

Comment:
why disable this?

Reply:
Oops~ IntelliJ IDEA can not sort imports properly when this line is enabled. So I commented it locally.
This should not be committed. I will revert it.